### DISCUSSION TITLE
How can I disable "Interactive Story Generation"

### DISCUSSION THREAD

**Discussion by MWhite-22** [storybookjs/storybook#28332] [Help]
### Summary

Is there a way to disable this feature?

We have over 100 developers and since upgrading to latest SB, this feature has caused an absolute explosion in "useless" stories that are nothing more than a tiny derivation change in another story.

**Comment by valentinpalkovic** [comment]
Duplicate of https://github.com/storybookjs/storybook/discussions/27219
